modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
If one is running the meta-analysis then beta.o always exists. If you are doing data assimilation then there are no sub-variables and the initial selection by variable just returns the vector of samples from the parameter.